actionview/lib/action_view/helpers/text_helper.rb

```diff
@@ -225,7 +225,10 @@ def excerpt(text, phrase, options = {})
       #
       #   pluralize(2, 'Person', locale: :de)
       #   # => 2 Personen
-      def pluralize(count, singular, plural = nil, locale: nil)
+      def pluralize(count, singular, options = {})
```

Comment:
Needs to deprecate `plural` as a positional argument. We should also use keyword arguments for the new version.

Would do:

```ruby
def pluralize(count, singular, deprecated_plural = nil, plural: nil, locale: I18n.locale)
  if deprecated_plural
    ActiveSupport::Deprecation.warn("Passing plural as a positional argument " \
      "is deprecated and will be removed in Rails 5.1. Use e.g. " \
      "pluralize(1, 'person', plural: 'people') instead.")
    plural ||= deprecated_plural
  end
  
  # ...
end
```